apps/ledger-live-mobile/package.json

```diff
@@ -158,6 +159,7 @@
     "react-native-extra-dimensions-android": "^1.2.5",
     "react-native-fast-crypto": "^2.2.0",
     "react-native-fast-image": "^8.5.11",
+    "react-native-fast-pbkdf2": "^0.3.1",
```

Comment:
could you explain where this library is used? I don't see it imported in LLM 🤔

Reply:
Hi! `@ton/crypto` depends on `@ton/crypto-primitives`, which in react-native depends on it, but it's not stated as a dependency. Here you have the line where is imported https://github.com/ton-org/ton-crypto-primitives/blob/f5c7482bf8d03ea39fa1668ec0fe458e0b516658/src/native/pbkdf2_sha512.ts#L12

Reply:
And `react-native` export is here https://github.com/ton-org/ton-crypto-primitives/blob/f5c7482bf8d03ea39fa1668ec0fe458e0b516658/package.json#L9